client/components/editor/ActivityContent/index.vue

```diff
@@ -161,6 +162,6 @@ export default {
 }
 
 .loader {
-  margin-top: 4.375rem;
+  margin-top: 1.25rem;
```

Comment:
You have a vertical movement now when the skeleton loader switches to containers (should look like the same sheet)